nodes/node_crate.lua

```diff
@@ -77,6 +77,7 @@ local store_digtron = function(pos, clicker, loaded_node_name, protected)
 	
 	local meta = minetest.get_meta(pos)
 	meta:set_string("crated_layout", layout_string)
+	meta:set_string("owner", clicker:get_player_name() or "")
```

Comment:
This would set the owner on unlocked crates, too, even if without effect at the moment.
Better use an `if protected then` guard here to avoid any mod interoperability issues.

Reply:
Related issue: #29